support/lobby/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<html><head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <title>Virtual World Framework lobby</title>
+
+  <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+  <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap-theme.min.css">
+  <link rel="stylesheet" href="index.bundle.css">
+
+</head><body>
```

Comment:
Interesting indentation scheme ... I would normally put `<body>` on its own line to signal the beginning of a new block.  But it looks like you've adopted more of an `Egyptian brackets` style.  I don't have strong opinions either way.  Just thought that I would comment.  :smile:

Reply:
Since there's almost always a `head` and a `body` and nothing else at the top, I don't see any need to preserve additional indention within `html` . It looks odd if you're looking for the strict tree, but I like that this way clearly identifies and separates the two sections.